website/docs/targeting/targeting-rule/user-condition.md

```diff
@@ -0,0 +1,196 @@
+
+### Comparison Value
+
+A string, a list of strings, a number, a semantic version, a list of semantic versions or a date value. Will be compared to the selected _Comparison attribute_ using the _Comparator_. The length of the _Comparison value_ is limited, and the limit depends on your subscription plan. See the [plan limits page](../../../subscription-plan-limits) for more details.
+
+### Comparator
+
+#### Text Comparators
+
+The following comparators expect the *Comparison attribute* to be a string value and the *Comparison value* to be a string or a list of strings.
+
+:::info
+Consider using Confidential text comparators if you plan to target users by their sensitive information, e.g.: email address or company domain.
+:::
+
+
+| Comparator                         | Description                                                                                     |
+| ---------------------------------- | ----------------------------------------------------------------------------------------------- |
+| EQUALS (cleartext)                 | Checks whether the comparison attribute equals the comparison value.                           |
+| NOT EQUALS (cleartext)             | Checks whether the comparison attribute is not equal to the comparison value.                  |
+| IS ONE OF (cleartext)              | Checks whether the comparison attribute is equal to any of the comparison values.              |
+| IS NOT ONE OF (cleartext)          | Checks whether the comparison attribute is not equal to any of the comparison values.          |
+| STARTS WITH ANY OF (cleartext)     | Checks whether the comparison attribute starts with any of the comparison values.              |
+| NOT STARTS WITH ANY OF (cleartext) | Checks whether the comparison attribute does not start with any of the comparison values.      |
+| ENDS WITH ANY OF (cleartext)       | Checks whether the comparison attribute ends with any of the comparison values.                |
+| NOT ENDS WITH ANY OF (cleartext)   | Checks whether the comparison attribute does not end with any of the comparison values.        |
+| CONTAINS ANY OF (cleartext)        | Checks whether the comparison attribute contains any comparison values as a substring.         |
+| NOT CONTAINS ANY OF (cleartext)    | Checks whether the comparison attribute does not contain any comparison values as a substring. |
+
+#### Confidential Text Comparators
+
+We recommend using confidential text comparators especially in case of frontend applications targeting users based on sensitive data (like email addresses, names, etc).
+In this case, the feature flag evaluation is performed using the SHA256 hashes of the values to ensure that the comparison values are not exposed. This can cause an increase in the size of the config.json file and the overall network traffic. It is recommended to use confidential comparators only when necessary.
```

Comment:
A "It is recommended to use confidential comparators only when necessary." szerintem nem igazán jó, szerintem a javaslat az, hogy használjanak mindig confidentialt, és ha esetleg network traffic para van, akkor consider using text comparators.